samples/bluetooth/direct_test_mode/README.rst

```diff
@@ -16,7 +16,7 @@ The sample uses Direct Test Mode to test the operation of the following features
 * packet error rate,
 * intermodulation performance.
 
-Test procedures are defined in the document `Bluetooth Low Energy RF PHY Test Specification`_: Document number RF-PHY.TS/5.0.0.
+Test procedures are defined in the document `Bluetooth Low Energy RF PHY Test Specification`_: Document number RF-PHY.TS.p15
```

Comment:
Please check if link update is also required after changing the document number

Reply:
Link is up to date leads to the most recent document